packages/data/abstract/prophecies/lib/src/json.dart

```diff
@@ -1,25 +1,25 @@
 import 'dart:convert';
 
-import '../prophecies_data_provider.dart';
 import 'package:prophecy_model/prophecy_model.dart';
+import 'prophecies.dart';
 
-class PropheciesJsonDataProvider implements PropheciesDataProvider {
+class PropheciesJson implements Prophecies {
```

Comment:
I will look up to it. A problem with the prophecies, we don't know how it will be used in future. Maybe we should not touch it.
For example, PropheciesJson is not even fully implemented for now.